experimental/packages/exporter-logs-otlp-grpc/src/index.ts

```diff
@@ -17,4 +17,4 @@
 /* eslint no-restricted-syntax: ["warn", "ExportAllDeclaration"] --
  * TODO: Replace wildcard export with named exports before next major version
  */
```

Comment:
@chancancode A late question on this one. I assume this eslint and TODO comment could now be removed, right?

Reply:
https://github.com/open-telemetry/opentelemetry-js/pull/5426 added to remove it.